frontend/src/ui/SearchableSelect.tsx

```diff
@@ -34,29 +34,51 @@ export const SearchableSelect = <T, >({
     ...props
 }: SearchableSelectProps<T>) => {
     const { t } = useTranslation();
+    const darkMode = useColorScheme().scheme === "dark";
 
     return <AsyncSelect
         loadOptions={loadOptions}
         formatOptionLabel={(option: T) => format(option, t)}
         cacheOptions
+        menuIsOpen // TODO: remove this line when PR is ready.
```

Comment:
This is here to be able to inspect the menu in dev tools. I guess it's not a big deal but there are some `border-radius` issues going on where the non-rounded corners of the first and last item overlap with the rounded menu border (also happens in light mode, but it's less apparent there). We could either try solving this with child-selectors or just keep it as is🤷 
<img width="209" alt="Bildschirmfoto 2023-06-12 um 12 57 48" src="https://github.com/elan-ev/tobira/assets/94838646/91b177ff-cf16-4369-b9df-c037395f359f">